setup.py

```diff
@@ -20,7 +20,7 @@
 from setuptools import find_packages
 from setuptools import setup
 
-version = '4.2.5.dev0'
+version = '4.3.0.dev0'
```

Comment:
Rebase to pick up that this is done already in #70?

Reply:
I missed changing setup.py in that PR, I just hit CHANGES.rst.

Reply:
That's why I usually avoid doing either of those changes until just before making the release.